applications/asset_tracker/src/main.c

```diff
@@ -385,16 +387,13 @@ static void send_agps_request(struct k_work *work)
 
 	LOG_INF("Sending A-GPS request");
 
-	err = nrf_cloud_agps_request_all();
+	err = gps_agps_request(agps_request, -1);
```

Comment:
can make -1 a human readable define in the agps api?

Reply:
Yes, how about ```GPS_SOCKET_INVALID```? Any suggestions?

Reply:
Evt. GPS_SOCKET_NOT_PROVIDED

Reply:
that's the one.